a3p-integration/README.md

```diff
@@ -33,37 +33,49 @@ make -C ../packages/deployment docker-build-sdk
 
 ## Run the integration test
 
-The `@agoric/synthetic-chain` test runner can be invoked through yarn.
+The `@agoric/synthetic-chain` test runner is invoked through NPM scripts.
 
 To build the agoric-3 synthetic chain images with the unreleased chain software upgrade:
 ```sh
-yarn synthetic-chain build
+yarn build
 ```
 
 To run the unit tests:
 ```sh
-yarn synthetic-chain test
-# or for an interactive session
-yarn synthetic-chain test --debug
+yarn test
+# or for an interactive session into a given proposal layer
+yarn test --debug -m "proposal-name"
 ```
 
 # Proposal structure
 
 ## Layering
 
-Proposal packages are under the `a3p-integration/proposals` folder, and are separate from each other, from the `a3p-integration` test environment, and from the enclosing `agoric-sdk` repository. They run inside the docker image, and cannot access SDK code. The are executed in lexical order.
+Proposal layers are packages under the `a3p-integration/proposals` folder, and are separate from each other, from the `a3p-integration` test environment, and from the enclosing `agoric-sdk` repository. They run inside the docker image, and cannot access SDK code. The are lower case, and executed in lexical order.
```

Comment:
Proposal layers are _defined by_ packages. (Each package can specify multiple layers, according to https://github.com/Agoric/agoric-3-proposals/tree/main/packages/synthetic-chain#design

If we were to talk about images instead of layers, there would still be two: use and test.

*They* are lower case (btw I intend have `doctor` check for that)

Reply:
when I say package here I mean a folder under `proposals` which has a `package.json`. Can one of those define multiple layers? How?

Reply:
Roughly. `prepare-`, `execute-`, `use-` are all build _stages_. We also tag a build targeting the `use-` stage as a `use-` image.

[Layers](https://docs.docker.com/build/guide/layers/,) are more numerous, 
> Each instruction in a Dockerfile roughly translates to an image layer